module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,39 @@
+#include "sort.hpp"
+
+#include <algorithm>
+#include <map>
+#include <vector>
+
+std::deque<std::string> lengthSort(std::forward_list<std::string> lst) {
```

Comment:
pass by reference 
```suggestion
std::deque<std::string> lengthSort(std::forward_list<std::string>& lst) {
```

Reply:
I think that it can be even const reference 
```suggestion
std::deque<std::string> lengthSort(const std::forward_list<std::string>& lst) {
```